src/clr-angular/data/datagrid/_properties.datagrid.scss

```diff
@@ -0,0 +1,36 @@
+// Copyright (c) 2016-2019 VMware, Inc. All Rights Reserved.
+// This software is released under MIT license.
+// The full license information can be found in LICENSE in the root directory of this project.
+
+@include exports('datagrid.properties') {
+  @if $clr-use-custom-properties == true {
+    :root {
+      --clr-datagrid-font-color: #{$clr-datagrid-font-color};
+      --clr-datagrid-default-border-color: var(--clr-color-neutral-400);
+      --clr-datagrid-icon-color: var(--clr-color-neutral-600);
+
+      --clr-datagrid-row-hover: var(--clr-color-neutral-200);
+      --clr-datagrid-row-hover-font-color: var(--clr-color-neutral-1000);
+      --clr-datagrid-action-toggle: var(--clr-color-neutral-700);
```

Comment:
just a note that from variable names `--clr-datagrid-row-hover` and `--clr-datagrid-action-toggle` don't make it obvious that they are colors. Maybe more so for `--clr-datagrid-action-toggle`. You don't have to change it, but if we have the liberty to I think it'd be nice to.

Reply:
I realized that this would make it hard for consumers to do "search and replace" to migrate from sass variable to custom properties. If that's preventing the name change could we consider marking for rename in 4.0?

Reply:
I can mark for rename. As it is, I was trying for a 1-to-1 parity between properties and variables.